docs/journalist/starting_client.rst

```diff
@@ -1,16 +1,74 @@
-Starting the SecureDrop client
+Starting the SecureDrop Client
 ==============================
 
 .. include:: ../includes/top-warning.rst
 
+Double-click on the **SecureDrop Workstation Launcher** desktop shortcut to
+launch the SecureDrop Client application.
+
+.. | screenshot_desktop-shortcut|
+
 Performing updates
 ------------------
 
+When you launch the application, it will first check for security
+updates. This can take several minutes.
+
+.. | screenshot_check_updates |
+
+If updates are found, you will be asked to apply them before launching the
+client. You have the option of completing this now, or at another time. Keep in
+mind that updates are important to security, and you should apply them the same
+day you receive the notification.
+
+It is important that, when you do click "Apply Updates," you allow the update
```

Comment:
Let's make this an `important` call-out, and clarify why (interrupting updates may cause system components to break).

Reply:
fixed in 5eda33f